applications/crossbar/src/cb_mobile_manager.erl

```diff
@@ -0,0 +1,94 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2010-2015, 2600Hz
+%%% @doc
+%%%
+%%% @end
+%%% @contributors
+%%%   Peter Defebvre
+%%%-------------------------------------------------------------------
+-module(cb_mobile_manager).
+
+-export([delete_account/1]).
+
+-include("./crossbar.hrl").
+
+-define(MOD_CONFIG_CAT, <<"mobile_manager">>).
+
+%%--------------------------------------------------------------------
+%% @public
+%% @doc
+%%
+%% @end
+%%--------------------------------------------------------------------
+delete_account(Context) ->
+    AccountId = cb_context:account_id(Context),
+    AuthToken = cb_context:auth_token(Context),
+
+    UrlString = req_uri([<<"accounts">>, AccountId]),
+    Headers = req_headers(AuthToken),
+
+    lager:debug("mobile_manager delete via ~s", [UrlString]),
+
+    Resp = ibrowse:send_req(UrlString, Headers, 'delete'),
+    handle_resp(Resp).
+
+%%%===================================================================
+%%% Internal functions
+%%%===================================================================
+
+
+%%--------------------------------------------------------------------
+%% @private
+%% @doc
+%%
+%% @end
+%%--------------------------------------------------------------------
+-spec handle_resp(ibrowse_ret()) -> 'ok'.
+handle_resp({'ok', "200", _, Resp}) ->
+    lager:debug("mobile_manager success ~s", [decode(Resp)]);
+handle_resp({'ok', Code, _, Resp}) ->
+    lager:warning("mobile_manager error ~p. ~s", [Code, decode(Resp)]);
+handle_resp(_Error) ->
+    lager:error("mobile_manager fatal error ~p", [_Error]).
+
+%%--------------------------------------------------------------------
+%% @private
+%% @doc
+%%
+%% @end
+%%--------------------------------------------------------------------
+-spec decode(string()) -> ne_binary().
+decode(JSON) ->
+    try wh_json:encode(JSON) of
```

Comment:
I truly don't understand the point of this function. In `handle_resp`, the last element in the tuple will be the response body as a string (I believe that's the default in ibrowse). So the first thing tried with that string is to encode it to JSON? Why not just log the response body directly above? The response is unused aside from logging it so why bother doing anything to it?